tests/postgres_tests/test_introspection.py

```diff
@@ -0,0 +1,26 @@
+from io import StringIO
+
+from django.core.management import call_command
+from django.test.utils import modify_settings
+
+from . import PostgreSQLTestCase
+
+
+@modify_settings(INSTALLED_APPS={'append': 'django.contrib.postgres'})
+class InspectDBTests(PostgreSQLTestCase):
+    def assert_field_type_in_model(self, model, field_outputs):
```

Comment:
Usually we camel case assertions to match the unittest style. Maybe assertFieldsInModel (considering field_outputs is a list).